bundles/org.openhab.binding.linktap/README.md

```diff
@@ -0,0 +1,240 @@
+
+### Bridge Configuration Parameters
+
+| Name                  | Type   | Description                                                                                                                                                    | Recommended Values | Required | Advanced |
+|-----------------------|--------|----------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------|----------|----------|
+| host                  | String | The hostname / IP address of the gateway device                                                                                                                |                    | Yes      | No       |
+| username              | String | The username if set for the gateway device                                                                                                                     |                    | No       | No       |
+| password              | String | The password if set for the gateway device                                                                                                                     |                    | No       | No       |
+| enableMDNS            | Switch | On connection whether the mDNS responder should be enabled on the gateway device                                                                               | ON                 | No       | Yes      |
+| enforceProtocolLimits | Switch | If ON data outside of the allowed ranges against the protocol will be logged and not sent                                                                      | ON                 | No       | Yes      |
+| enableJSONComms       | Switch | OFF by default for backwards compatibility, if using up to date firmware with no other local network applications set to ON, for more efficient communications | ON                 | No       | Yes      |
+
+
+**NOTE** When enableMDNS is enabled, upon connection to the gateway option "Enable mDNS responder" is switched on 
+
+### LinkTap Device Configuration Parameters
+
+It is recommended to use the Device Id, for locating devices. This can be found in the LinkTap mobile application under 
+Settings->TapLinker / ValveLinker, e.g.
+
+- ValueLinker_1 (D71BC52F004B1200_1-xxxx)
+  - has Device Id "ValveLinker_1"
+  - has Device Name D71BC52F004B1200_1
+
+| Name         | Type   | Description                                                           | Recommended Values | Required | Advanced |
+|--------------|--------|-----------------------------------------------------------------------|--------------------|----------|----------|
+| deviceId     | String | The Device Id for the device under the gateway                        |                    | Yes      | No       |
+| deviceName   | String | The name allocated to the device by the app. (Must be unique if used) |                    | Yes      | No       |
```

Comment:
I have updated with a note to explain this. One or the other need's to be provided. If users provide both then in case of typo's the system can locate via the other for example. I personally like to use ID's but for inexperienced users if they name for example device1 as Baskets, then setting the device name to Baskets makes it much easier than trying to cross type the long id in to a text file. Hopefully the updates make it clearer.